src/ploneorg/addonlisting/browser/templates/addonfolder_view.pt

```diff
@@ -1,40 +1,169 @@
-<html xmlns="http://www.w3.org/1999/xhtml"
-      xmlns:metal="http://xml.zope.org/namespaces/metal"
-      xmlns:tal="http://xml.zope.org/namespaces/tal"
-      xmlns:i18n="http://xml.zope.org/namespaces/i18n"
-      i18n:domain="ploneorg.addonlisting"
-      metal:use-macro="context/main_template/macros/master">
-<body>
-  <metal:content-core fill-slot="content-core">
 
-  <metal:addonlist define-macro="addonlist">
-  <table class="listing"
-         id="AddOns">
-    <thead>
-      <tr>
-        <th>Title</th>
-        <th>Description</th>
-      </tr>
-    </thead>
-    <tbody>
-      <tr tal:repeat="addon view/all_addons" tal:condition="view/all_addons">
-        <td tal:content="addon/title">
-            Title
-        </td>
-        <td tal:content="addon/description">
-            Description
-        </td>
-      </tr>
-      <tr tal:condition="not:view/all_addons">
-        <td colspan="2">
+<html
```

Comment:
There must be something wrong with this file, My linter complains the indention, which itself is no problem, but if I beautify / auto format the file I get a strange indention at the end. There must be some elements that are not correctly closed.

Please have a look at that.